examples/fakegroundstation/ground_station_service.py

```diff
@@ -0,0 +1,136 @@
+import time
+from concurrent import futures
+
+import grpc
+from google.protobuf.timestamp_pb2 import Timestamp
+
+from stellarstation.api.v1.groundstation import groundstation_pb2_grpc
+from stellarstation.api.v1.groundstation import groundstation_pb2
+
+
+SECONDS_BEFORE_PLAN_START = 10
+PLAN_DURATION_SECONDS = 600
+CURRENT_PLAN_ID = "3"
+
+
+class GroundStationServiceServicer(groundstation_pb2_grpc.GroundStationServiceServicer):
+    def ListPlans(self, request, context) -> groundstation_pb2.ListPlansResponse:
```

Comment:
Use snake case for Python function name.

Reply:
These are overriding the methods of the `groundstation_pb2_grpc.GroundStationServiceServicer` stub.

Reply:
Oh, I see. Thanks for the clarification.